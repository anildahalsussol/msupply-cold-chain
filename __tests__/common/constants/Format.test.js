import { FORMAT } from '~constants';

describe('Constants: FORMAT', () => {
  it('Matches snapshot', () => {
    expect(FORMAT).toMatchSnapshot();
  });
});
