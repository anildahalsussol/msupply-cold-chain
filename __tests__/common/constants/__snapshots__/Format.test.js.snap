// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: FORMAT Matches snapshot 1`] = `
Object {
  "DATE": Object {
    "HOUR_WITH_PERIOD": "ha",
    "STANDARD_DATE": "DD/MM/YYYY",
    "STANDARD_TIME": "HH:mm",
  },
}
`;
